src/main/java/org/wso2/carbon/connector/azure/storage/util/AzureConstants.java

```diff
@@ -28,6 +28,7 @@
     public static final String CONTAINER_NAME = "containerName";
     public static final String FILE_NAME = "fileName";
     public static final String FILE_PATH = "filePath";
+    public static final String BLOBCONTENTTYPE = "blobContentType";
```

Comment:
Shall we have `_` as below, to improve readability?
```suggestion
    public static final String BLOB_CONTENT_TYPE = "blobContentType";
```